pkg/api/registry/registry.proto

```diff
@@ -22,61 +22,44 @@ message Destination {
     uint32 weight = 2;
 }
 
-message NetworkServiceManager {
-    string name = 1;
-    string url = 2;
-    google.protobuf.Timestamp expiration_time = 3;
-    string state = 4;
-}
-
 message NetworkServiceEndpoint {
     string name = 1;
-    string payload = 2;
-    string network_service_name = 3;
-    string network_service_manager_name = 4;
-    map<string, string> labels = 5;
-    string state = 6;
-
+    repeated string network_service_name = 2;
+    map<string, string> labels = 3;
     // Endpoint callback address, could be any of unix/tcp/callback schema ones.
-    string url = 7;
+    string url = 4;
```

Comment:
Should we consider `repeated string url = 4` so when endpoint is registered from Nsmgr, it will have two urls, one is url of manager and one it internal one?

Reply:
Hmm.. probably then we should use the map
```
map<string, string>  urls
```
to get rid of problems with URL order convention,
Then we can use it something like
```
nse_url: callback://addr
nsm_url: tcp://addr
extra_url: ....
```
@edwarnicke , @haiodo What do you think?

Reply:
If we have scenario when it could be used map is better structured in compare to list. We we not have such scenario when separate field is ok.